Makefile

```diff
@@ -503,12 +501,6 @@ test: generate fmt vet manifests build-cli-mocks ## Run tests
 test-cli: build-cli-mocks ## Run tests
 	$(GO) test ./pkg/v1/cli/... ./pkg/v1/auth/... ./pkg/v1/builder/... ./pkg/v1/config/... ./pkg/v1/encoding/... ./pkg/v1/grpc/...
 
-fmt: tools ## Run goimports
-	$(GOIMPORTS) -w -local github.com/vmware-tanzu ./
```

Comment:
We should keep this target as it's actually sorting (writing) imports and enforcing a `-local` rule. The linter only checks if imports are sorted. This is handy when linter complains and you can just run `make fmt` ([example](https://github.com/vmware-tanzu/tanzu-framework/pull/2530#issuecomment-1146858106)).

Reply:
Done!